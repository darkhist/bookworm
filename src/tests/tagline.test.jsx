import React from 'react';
import { shallow } from 'enzyme';

import Tagline from '../components/tagline/tagline';

describe('Tagline', () => {
  it('should render without crashing', () => {
    const tagline = shallow(<Tagline />);
    expect(tagline).toMatchSnapshot();
  });

  it('should return some helpful text', () => {
    const tagline = shallow(<Tagline />);
    expect(tagline.html()).toContain('Try searching by title or author');
  });
});
