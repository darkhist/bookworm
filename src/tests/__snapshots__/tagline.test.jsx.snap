// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tagline should render without crashing 1`] = `
<styled.p>
  Discover new books!
  <br />
  Try searching by title or author!
</styled.p>
`;
